roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -138,6 +134,8 @@ def run(self):
                 blocked=blocked_msg if self.registries["blocked"] else "",
                 unreachable=unreachable_msg if unreachable else "",
             )
+            if six.PY2:
```

Comment:
Is this part necessary?  Should already be utf8 with the u"" syntax, I believe.

Reply:
Yeah, other parts are not expecting the message to be unicode - namely `zz_failure_summary.py` callback_plugin and `register_failure` in `openshift_checks/__init__.py`